cypress/e2e/11-inline-expression-editor.cy.ts

```diff
@@ -0,0 +1,66 @@
+import { WorkflowPage as WorkflowPageClass } from '../pages/workflow';
+
+const WorkflowPage = new WorkflowPageClass();
+
+describe('Inline expression editor', () => {
+	before(() => {
+		cy.task('reset');
+		cy.skipSetup();
+	});
+
+	beforeEach(() => {
+		WorkflowPage.actions.visit();
+		WorkflowPage.actions.addInitialNodeToCanvas('Manual Trigger');
+		WorkflowPage.actions.addNodeToCanvas('Hacker News');
+		WorkflowPage.actions.openNodeNdv('Hacker News');
+		WorkflowPage.actions.openInlineExpressionEditor();
```

Comment:
Is there a way to move the content of `beforeEach` to `before` so the tests share setup and so run faster? I tried to but the run fails after the first test, redirecting to `about:blank`. Or is it best practice to keep all tests in a suite running from scratch?

Reply:
I guess it depends on what you are trying to do, but I guess having a clean state before each test is in line with cypress guidelines of writing independent tests.